### PR TITLE
Prevent LDAP Password to appear in log file

### PR DESCRIPTION
**Pull request by voroyam**
If connection to LDAP server is lost, the LDAP password is shown in the stack trace.

This should prevent this from happening.

### CODE REVIEW

**Comment by cdamken** [comment]
@tomneedham @PVince81 @butonic Could you please review this?

File: lib/private/Log.php [hunk 1]
```diff
@@ -101,7 +101,14 @@ class Log implements ILogger {
 		'decrypt',
 
 		//LoginController
-		'tryLogin'
+		'tryLogin',
+
+		//bind
```
Comment (PVince81) [inline lib/private/Log.php]: these comments look useless...

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by PVince81** [comment]
@voroyam please backport to stable10

File: lib/private/Log.php (hunk 1 above)
Comment (voroyam) [reply lib/private/Log.php]: They are, any suggestions how to improve them?

**Comment by voroyam** [comment]
@PVince81 don't know how to backport, already told @tom this.

**Comment by PVince81** [comment]
stable10: https://github.com/owncloud/core/pull/32592

**Comment by patrickjahns** [comment]
Regarding the comments about useless string - suggestion, mention that the two methods `bind` // `invokeLDAPMethod` result from `user_ldap`.

```
// user_ldap app 
'bind'
'invokeLDAPMethod'
```

**Pull request merged**